src/pages/sa-financial-integrator/index.js

```diff
@@ -0,0 +1,129 @@
+import { useContext } from 'react'
+import toast from 'react-hot-toast'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { useResourceQuery } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { useForm } from 'src/hooks/form'
+import FormShell from 'src/components/Shared/FormShell'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { DataGrid } from 'src/components/Shared/DataGrid'
+import { GeneralLedgerRepository } from 'src/repositories/GeneralLedgerRepository'
+import { ControlContext } from 'src/providers/ControlContext'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import WindowToolbar from 'src/components/Shared/WindowToolbar'
+import { Formik } from 'formik'
+import { SaleRepository } from 'src/repositories/SaleRepository'
+import { FinancialRepository } from 'src/repositories/FinancialRepository'
+
+const FinancialIntegrators = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+
+  const getGridData = async () => {
+    const res = await getRequest({
+      extension: SaleRepository.FinancialIntegrators.qry,
+      parameters: `_filter=`
+    })
+    formik.setValues({
+      ...formik.values,
+      rows: res.list.map(({ ...rest }, index) => ({
+        id: index + 1,
+        ...rest
+      }))
+    })
+  }
+
+  const { labels: labels, maxAccess } = useResourceQuery({
+    queryFn: getGridData,
+    datasetId: ResourceIds.FinancialIntegrators
+  })
+
+  const { formik } = useForm({
+    maxAccess,
+    enableReinitialize: true,
+    validateOnChange: true,
+    initialValues: {
+      rows: []
```

Comment:
shouldn't you add cgId, fgId and sameNumber in the rows in the initial values as null?